src/opnsense/mvc/app/controllers/OPNsense/Proxy/Api/SettingsController.php

```diff
@@ -240,4 +241,53 @@ public function fetchRBCronAction()
 
         return $result;
     }
+
+    /**
+     * get action
+     * @return array
+     */
+    public function getAction()
+    {
+        // define list of configurable settings
+        $result = array();
+        if ($this->request->isGet()) {
+            $mdlProxy = $this->getModel();
+            $mdlProxy->forward->acl->whiteList = IDN::decode($mdlProxy->forward->acl->whiteList);
```

Comment:
better to use standard php functions http://www.php.net/manual/en/function.idn-to-ascii.php and wrap it into a method for this class if you need to split the strings.